src/test/scala/sigmastate/utxo/BasicOpsSpecification.scala

```diff
@@ -276,7 +280,12 @@ class BasicOpsSpecification extends SigmaTestingCommons {
         |}""".stripMargin,
       {
         val data = GetVar(dataVar, dataType).get
-        Exists(data, 21, EQ(SelectField(TaggedVariable(21, STuple(SCollection(SByte), SLong)), 2), LongConstant(10)))
+        OR(
```

Comment:
@greenhat Why `exists` is transformed to `map`? Where is this happens?

Reply:
@ergomorphic I suppose it starts here - http://github.com/ScorexFoundation/sigmastate-interpreter/blob/c8b1773de7996455f7ac5e7279204b32f3fb3d0f/src/main/scala/sigmastate/eval/RuntimeCosting.scala#L963 
and then - http://github.com/ScorexFoundation/sigmastate-interpreter/blob/c8b1773de7996455f7ac5e7279204b32f3fb3d0f/src/main/scala/sigmastate/lang/SigmaBuilder.scala#L269